atomic

```diff
@@ -514,6 +518,18 @@ def create_parser(help_text):
                      help=_("Only display container IDs"))
     add_opt(pss)
 
+    # atomic sign
+    signer = ATOMIC_CONFIG.get('default_signer', None)
+    sig_store = ATOMIC_CONFIG.get('default-sigstore-path', "/var/lib/atomic/containers")
+    signp = subparser.add_parser("sign", help="Sign an image",
+                                 epilog="Create a signature for an image which can be used later to verify it.")
+    signp.set_defaults(_class=Sign, func="sign")
+    signp.add_argument("images", nargs="*", help=_("images to sign"))
+    signp.add_argument("--sign_key", dest="sign_key", default=signer,
+                       help=_("Name of the signing key. Currently %s, default can be defined in /etc/atomic.conf" % signer))
+    signp.add_argument("-o", "--output", dest="output", default=sig_store,
+                       help=_("Path to store the signature ({})".format(sig_store)))
```

Comment:
See the discussion around users of this option, `default-sigstore-path` and `--output` are not doing the same thing.  (The option help could still mention that default, but using something like `(instead of creating a file within the directory structure under {})`—if it isn’t too long for `--help`.)